entregas/rasillaJose/003-ListaSimple/Node.java

```diff
@@ -0,0 +1,33 @@
+public class Node {
+
+    private String value;
+    private Node next;
+    private boolean isLast = true;
+
+
+    public boolean getisLast(){
+        return isLast;
+    }
```

Comment:
¿No haría lo mismo `return this.next==null` y nos ahorramos una variable?. Bueno, y el nombre (horroroso) de getisLast() que pasa a ser isLast()?